package.json

```
@@ -35,9 +35,24 @@
       {
         "command": "iliazeus.vscode-ansi.showPretty",
         "title": "Show as ANSI",
-        "enablement": "editorActive"
+        "icon": "$(open-preview)"
+      },
+      {
+        "command": "iliazeus.vscode-ansi.showPrettyToSide",
+        "title": "Show as ANSI to the Side",
```

Should the language more closely match how markdown works?

```suggestion
        "title": "Open Preview to the Side",
```

---

It makes sense, I guess. I didn't really like the word "preview" initially, but I'm not a native speaker.

---

Addressed these issues in 8706dc2.